backend/src/test/java/com/daedan/festabook/place/controller/PlaceAnnouncementControllerTest.java

```diff
@@ -32,4 +43,43 @@ class PlaceAnnouncementControllerTest {
     void setUp() {
         RestAssured.port = port;
     }
+
+    @Nested
+    class deleteByPlaceAnnouncementId {
+
+        @Test
+        void 성공() {
+            // given
+            Festival festival = FestivalFixture.create();
+            festivalJpaRepository.save(festival);
+
+            Place place = PlaceFixture.create(festival);
+            placeJpaRepository.save(place);
+
+            PlaceAnnouncement placeAnnouncement = PlaceAnnouncementFixture.create(place);
+            placeAnnouncementJpaRepository.save(placeAnnouncement);
+
+            // when & then
+            RestAssured
+                    .given()
```

Comment:
given은 없으니 when으로 사용해도 될 것 같아요.
```suggestion
                    .when()
```

Reply:
둘다 넣겠습니다.!